tests/providers/test_misc.py

```diff
@@ -455,6 +456,12 @@ def test_dsv_csvwriter_kwargs(self, faker):
             for args, kwargs in mock_writer.call_args_list:
                 assert kwargs == test_kwargs
 
+    def test_xml(self, faker):
+        try:
+            xml.etree.ElementTree.fromstring(faker.xml())
+        except xml.etree.ElementTree.ParseError:
+            raise AssertionError("The xml format is invalid")
```

Comment:
can we always capitalize `XML` and title-case `Python` in strings?

Reply:
@fcurella
Can you please provide more details? I didn't quite understand what the question was

Reply:
```suggestion
            raise AssertionError("The XML format is invalid.")
```